arduino-adafruit-samd-feather-m0.js

```diff
@@ -9,6 +9,8 @@
  */
 function initTasks(gulp, options) {
   var all = require('./all.js')(options);
+  // stick config into gulp object
+  gulp.config = all.getConfig();
```

Comment:
it's ok for now, just need to find a way to prevent config.h from being stored in GitHub...